frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -0,0 +1,140 @@
+import classNames from 'classnames';
+import { compose } from 'redux';
+import { useTranslation } from 'react-i18next';
+import { ChartDonut, ChartLabel } from '@patternfly/react-charts';
+import { usePrometheusQueries } from '@console/shared/src/components/dashboard/utilization-card/prometheus-hook';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import { humanizeBinaryBytes } from '@console/internal/components/utils';
+import { getInstantVectorStats } from '@console/internal/components/graphs/utils';
+import { CAPACITY_INFO_QUERIES } from '../../../../constants/queries';
+import './raw-capacity-card.scss';
+
+const queries = (() => Object.values(CAPACITY_INFO_QUERIES))();
+const colorScale = ['#0166cc', '#d6d6d6'];
+
+// Enchance instantVectorStats to directly parse the values (else loading state won't be accurate)
+const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
+
+const RawCapacityCard: React.FC = React.memo(() => {
+  const [values, loading, loadError] = usePrometheusQueries(queries, parser as any);
+  const { t } = useTranslation();
+
+  const totalCapacityMetric = values?.[0];
+  const usedCapacityMetric = values?.[1];
+
+  const totalCapacity = humanizeBinaryBytes(totalCapacityMetric);
+  const availableCapacity = humanizeBinaryBytes(
+    totalCapacityMetric - usedCapacityMetric,
+    null,
+    totalCapacity?.unit,
+  );
+  const usedCapacity = humanizeBinaryBytes(usedCapacityMetric, null, totalCapacity?.unit);
+
+  // Adjusted units
+  const usedCapacityAdjusted = humanizeBinaryBytes(usedCapacityMetric);
+  const availableCapacityAdjusted = humanizeBinaryBytes(totalCapacityMetric - usedCapacityMetric);
+
+  const donutData = [
+    { x: 'Used', y: usedCapacity.value, string: usedCapacityAdjusted.string },
+    {
+      x: 'Available',
+      y: availableCapacity.value,
+      string: availableCapacityAdjusted.string,
+    },
+  ];
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>{t('ceph-storage-plugin~Raw Capacity')}</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody className="ceph-raw-usage__container">
+        {!loading && !loadError && (
+          <>
+            <div className="ceph-raw-usage__item ceph-raw-usage__legend">
+              <ChartLegend
+                fill={colorScale[0]}
+                title="Used"
+                text={usedCapacityAdjusted.string}
+                titleClassName="ceph-raw-card-legend__title--pad"
+              />
+              <ChartLegend
+                fill={colorScale[1]}
+                title="Available"
```

Comment:
externalize this too
```suggestion
                title="Available"
```